### PR TITLE
update stable diffusion notebooks according the latest API changes

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

File: notebooks/225-stable-diffusion-text-to-image/225-stable-diffusion-text-to-image.ipynb [hunk 1]
```diff
@@ -80,7 +80,7 @@
    "source": [
     "from diffusers import StableDiffusionPipeline\n",
     "\n",
-    "pipe = StableDiffusionPipeline.from_pretrained(\"prompthero/midjourney-v4-diffusion\")"
+    "pipe = StableDiffusionPipeline.from_pretrained(\"prompthero/openjourney\", force_download=True)"
```
Comment (eaidova) [inline notebooks/225-stable-diffusion-text-to-image/225-stable-diffusion-text-to-image.ipynb:83]:
```suggestion
    "pipe = StableDiffusionPipeline.from_pretrained(\"prompthero/openjourney\")"
```

**Comment by raymondlo84** [comment]
@adrianboguszewski  & @AnishaUdayakumar  please again approve and validate these changes.

**Review by AnishaUdayakumar — COMMENTED** [review]
#691 is resolved with this fix.

**Review by AnishaUdayakumar — COMMENTED** [review]
Conversion of the ONNX models is throwing an error each time with a "cannot schedule new futures after shutdown" message.
Text Encoder - success
Unet - restart kernal
Autoencoder (VAE) - again restart kernel after UNET conversion

File: notebooks/225-stable-diffusion-text-to-image/225-stable-diffusion-text-to-image.ipynb [hunk 2]
```diff
@@ -179,18 +174,21 @@
...
-   "cell_type": "markdown",
-   "metadata": {},
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "PscvJUXf_hpm"
+   },
+   "outputs": [],
    "source": [
     "### U-net\n",
```
Comment (AnishaUdayakumar) [inline notebooks/225-stable-diffusion-text-to-image/225-stable-diffusion-text-to-image.ipynb:193]: Thanks for the quick fix on the mo @eaidova.
Request to include this content in a markdown cell.

Comment (eaidova) [reply notebooks/225-stable-diffusion-text-to-image/225-stable-diffusion-text-to-image.ipynb]: thank you, updated

**Review by AnishaUdayakumar — APPROVED** [review]

**Review by adrianboguszewski — APPROVED** [review]
It's working now

**Comment by dankwartrustow** [comment]
It looks like the notebook is still failing in configure inference pipeline sections. 

`unet_model = core.compile_model(UNET_OV_PATH, 'AUTO')` produces the error: "RuntimeError: Model file unet.xml cannot be opened!"

Two cells down where we run the command: 
`ov_pipe = OVStableDiffusionPipeline(
    tokenizer=tokenizer,
    text_encoder=text_enc,
    unet=unet_model,
    vae_encoder=vae_encoder,
    vae_decoder=vae_decoder,
    scheduler=lms
)`

It produces the error: "NameError: name 'unet_model' is not defined"

**Pull request merged**